### PR TITLE
[FLAPI-2040] Fix Duffel Payments class method

### PR DESCRIPTION
**Pull request by jesse-c**
Add missing `self` being passed. Instance methods should take a "self" parameter.

### CODE REVIEW

File: duffel_api/api/duffel_payments/payment_intents.py [hunk 1]
```diff
@@ -53,14 +53,14 @@ def __init__(self, client):
         self._amount = None
         self._currency = None
 
-    def _validate_payment_keys(payment_details):
+    def _validate_payment_keys(self, payment_details):
```
Comment (sgerrand) [inline duffel_api/api/duffel_payments/payment_intents.py:56]: This is a static method which I forgot to tag as such. I don't think that there's a need to make it an instance method.

**Review by sgerrand — COMMENTED** [review]
Suggest adding the appropriate decoration for a static method instead.

File: duffel_api/api/duffel_payments/payment_intents.py (hunk 1 above)
Comment (nlopes) [reply duffel_api/api/duffel_payments/payment_intents.py:56]: Yeah, this just needs the `@staticmethod`, not `self`.

Comment (sgerrand) [reply duffel_api/api/duffel_payments/payment_intents.py]: I've added the missing decorator in 896f664. Apologies for missing it previously!

**Review by sgerrand — APPROVED** [review]
LGTM. 👍

**Comment by jesse-c** [comment]
Hahah

Ah yes, that's a better solution.

**Pull request merged**